ACE/ace/Shared_Memory_Pool.cpp

```diff
@@ -123,7 +123,7 @@ ACE_Shared_Memory_Pool::commit_backing_store_name (size_t rounded_bytes,
                       "exceeded max number of segments = %d, base = %u, offset = %u\n",
                        counter,
                        this->base_addr_,
-                       offset),
+                       static_cast<unsigned int>(offset)),
                       -1);
```

Comment:
If there is a conversion warning shouldn't also a macro be used for the format specifier for offset instead of casting it to a 32bit value?